esm_runscripts/slurm.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, config):
         folder = config["general"]["thisrun_scripts_dir"]
         self.filename = "hostfile_srun"
         self.path = folder + "/" + self.filename
+        self.folder = folder + "/../work/" 
```

Comment:
Why is the work folder needed here? Is it not possible to use the same logic as with the `hostfile_srun`?